### DISCUSSION TITLE
Can someone explain how to add multi-user support to a apple silicon installation of brew

### DISCUSSION THREAD

**Discussion by shaunsingh** [Homebrew/discussions#424] [Everyday usage]
Basically the title, someone told me I need to edit BREW-PREFIX but I don't understand how to do that, and all the online guides explain only how to do it for /usr/local. I would really really love if someone could just make paste some terminal commands that I need to run, thank you in advance and sorry if I'm being picky.

**Comment by SMillerDev** [comment]
> If you need to run Homebrew in a multi-user environment, consider creating a separate user account especially for use of Homebrew.

I don't see why you'd need to edit brew prefix. But either way multi-user setups aren't supported. So if you can't figure out how to create one, I'm sure you're going to have a very bad time using one

**Reply by shaunsingh** [reply]
Ah, I heard it was good practice to have your admin account seperate from your main, and so I thought it would be a good idea to split it up 

do you know of any way to use brew through the other account in that case? Like logging in from the terminal

**Reply by SMillerDev** [reply]
You can use `sudo` to do that. Again, this is an unsuported configuration and you're making your install more difficult than it needs to be.
